### PR TITLE
Remove Window clashing port check

### PR DESCRIPTION
**Pull request by hardillb**
Given we've changed the default port numbers this section should no longer be required

### CODE REVIEW

**Comment by Steve-Mcl** [comment]
Hey Ben, I was going to pick this up next (after audit log stuff)

There is a touch more than just removing that section - there are a few changes to sync from forge/docs/install/local/readme 

Pretty much everything from [prerequisites](https://github.com/flowforge/flowforge/blob/main/docs/install/local/README.md#prerequisites) to the bottom needs copying to this readme.

I am about 75% done with audit logs if you want I can pick this up next?

**Comment by hardillb** [comment]
Np, can close this then. I was just looking after updating the mosquitto docs and notices

**Comment by Steve-Mcl** [comment]
no its fine Ben, leave it open, i will put updates in here.

**Comment by Steve-Mcl** [comment]
@hardillb - please have a quick glance - I synced it up - a quick glance there is nothing silly pls.

File: README.md [hunk 1]
```diff
@@ -63,25 +65,30 @@ You will also need to install the appropriate build tools.
     mkdir c:\flowforge
     ```
 
-2. Download the Installer zip file from https://github.com/flowforge/installer/releases
+2. Download the latest [Installer zip file](https://github.com/flowforge/installer/releases/latest) into a temporary location.
```
Comment (hardillb) [inline README.md:68]: We can link directly to the file with

`https://github.com/flowforge/installer/releases/latest/download/flowforge-installer.zip`

Comment (hardillb) [reply README.md:68]: (once the next release is cut)

**Review by hardillb — COMMENTED** [review]
Looks good, I don't think I can approve as I opened the PR

**Review by Steve-Mcl — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (hardillb) [reply README.md:68]: Are we going to change this after Thursday?

Comment (hardillb) [reply README.md:68]: https://github.com/flowforge/installer/issues/61

**Pull request merged**